fastify.d.ts

```diff
@@ -149,7 +147,7 @@ declare namespace fastify {
   interface ServerOptions {
     ignoreTrailingSlash?: boolean,
     bodyLimit?: number,
-    logger?: pino.LoggerOptions | boolean,
+    logger?: boolean,
```

Comment:
I think this should be `any`. 
```suggestion
    logger?: any,
```